example/package.json

```diff
@@ -21,12 +22,22 @@
   "devDependencies": {
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "babelify": "^8.0.0",
```

Comment:
This was complaining that I needed babel-core. Do we need to add that?

Reply:
Whoops, yes. It's a peer dependency of `babelify`. Will update.